packages/x-scheduler/src/primitives/utils/SchedulerStore/SchedulerStore.ts

```diff
@@ -316,7 +326,8 @@ export class SchedulerStore<
         // TODO: Issue #19440 + #19441 - Allow to edit all events or only this event.
         scope = await chooseRecurringEventScope();
       } else {
-        scope = 'this-and-following';
+        // TODO: Issue #19766 - Let the user choose the scope via UI.
+        scope = 'all';
```

Comment:
If you think #19766 can take more than a couple of week before being merged, then I would do this:


```suggestion
        scope = window.SCHEDULER_RECURRING_EDITING_SCOPE ?? 'all';
```

That way you can easily test the various behaviors without touching the code.

Reply:
That's a good point! Done ✅